README.md

```diff
@@ -1,4 +1,5 @@
 # demo-repo
 User A makes edits/alterations
 User B makes edits/alterations
-User C makes edits on a branch!
\ No newline at end of file
+User C makes edits on a branch!
+I am making some more changes.
```

Comment:
This looks wrong! I think it's giong to break production. You are fired. 

A suggestion is to refactor this bla blah blah.